generators/client-2/templates/src/main/webapp/app/account/activate/_activate.state.ts

```diff
@@ -1,23 +1,20 @@
-ActivateStateConfig.$inject = ['$stateProvider'];
+import { ActivateComponent } from "./activate.component";
 
-export function ActivateStateConfig($stateProvider) {
-    $stateProvider.state('activate', {
-        parent: 'account',
-        url: '/activate?key',
-        data: {
-            authorities: [],
-            pageTitle: 'activate.title'
-        },
-        views: {
-            'content@': {
-                template: '<activate></activate>'
-            }
-        },
-        resolve: {
-            translatePartialLoader: ['$translate', '$translatePartialLoader', function ($translate, $translatePartialLoader) {
-                $translatePartialLoader.addPart('activate');
-                return $translate.refresh();
-            }]
-        }
-    });
-}
+export const activateState = {
+    name: 'activate',
+    parent: 'account',
+    url: '/activate?key',
+    data: {
+        authorities: [],
+        pageTitle: 'activate.title'
+    },
+    views: {
+        'content@': { component: ActivateComponent }
+    },
+    resolve: {
```

Comment:
resolve syntax for ng2 has changed, but it is supports ng1 style.  
I didn't migrate any resolves since mostly ng1 services are being utilized.